specification/eventgrid/data-plane/Microsoft.SignalRService/stable/2018-01-01/SignalRService.json

```diff
@@ -0,0 +1,59 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2018-01-01",
+        "title": "Schema of Azure SignalR Service events published to Azure Event Grid",
+        "description": "Describes the schema of the Azure SignalR Service events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+    },
+    "paths": {},
+    "definitions": {
+        "SignalRServiceClientConnectionConnectedEventData": {
+            "description": "Schema of the Data property of an EventGridEvent for a Microsoft.SignalRService.ClientConnectionConnected event.",
+            "type": "object",
+            "properties": {
+                "timestamp": {
+                    "description": "The time at which the event occurred.",
+                    "type": "date-time"
```

Comment:
please take a look at type and format for "date-time" https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types